build/Settings.cs

```diff
@@ -56,5 +57,76 @@ private static string config(string @default = null, [CallerMemberName] string k
         public static string CommitId => config(null, "Build.SourceVersion") ?? config("N/A", "APPVEYOR_REPO_COMMIT");
 
         public static string BuildArtifactsStorage => config(null);
+
+        public class SignInfo
+        {
+            public static string AzureSigningConnectionString => config(null, "AzureBlobSigningConnectionString");
+            public static readonly string AzureToSignContainerName = "azure-functions-cli";
+            public static readonly string AzureSignedContainerName = "azure-functions-cli-signed";
+            public static readonly string AzureSigningJobName = "signing-jobs";
+            public static readonly string ToSignZipName = $"{BuildNumber}-authenticode.zip";
+            public static readonly string ToSignThirdPartyName = $"{BuildNumber}-third.zip";
+            public static readonly string ToSignDir = "unsigned";
+            public static readonly string SignedDir = "signed";
+            public static readonly string Authenticode = "SignAuthenticode";
+            public static readonly string ThirdParty = "Sign3rdParty";
+            public static readonly string[] RuntimesToSign = new[] {"min.win-x86"};
+            public static readonly string[] filterExtenstionsSign = new[] { ".json", ".spec", ".cfg", ".pdb", ".config", ".nupkg", ".py" };
+
+            public static readonly string[] authentiCodeBinaries = new[] {
+                "DurableTask.AzureStorage.Internal.dll",
+                "DurableTask.Core.Internal.dll",
+                "func.deps.json",
+                "func.dll",
+                "func.dll.config",
+                "func.exe",
+                "func.pdb",
+                "func.runtimeconfig.json",
```

Comment:
non-executables like `.json`, `.config` etc don't have to be signed. executable scripts like `.js` and `.py` do need to be signed.

Reply:
The `.py` files in distlib were failing with the third party signing scripts, and it sounded like we neede d the `.dll`s to be signed asap, so for now, I ended us ignoring `.py` files.

Reply:
Just thought it was a bit misleading for .json and .config files to be in the list, but not a big deal.

Reply:
I think you are right, I will remove those files from the list

Reply:
I updated the PR and removed those files.